woocommerce/woocommerce
# add: partial spotlight for shipping smart defaults tour

Follow up PR to https://github.com/woocommerce/woocommerce/pull/33688 , part #2 of 33365

In the previous PR, the spotlight for steps 1 and 2 of the shipping smart defaults spotlight tour was only able to highlight a single table cell or the entire table, as @automattic/tour-kit does not have the ability to reference multiple elements for spotlighting.

This PR introduces an element that floats an invisible, non-interactive div above the indicated table cells so that we can highlight just the first two columns and the last column of the shipping zones settings table for steps 1 and 2, respectively.


### All Submissions:


### Changes proposed in this Pull Request:

Introduced an element that floats an invisible, non-interactive div above the indicated table cells for steps 1 and 2 of the shipping smart defaults tour.

Closes #33365 .

### How to test the changes in this Pull Request:

As the previous PR has already tested all the combinations of flows, we can focus only on steps 1 and 2 of the tour which are the same for all flows.

1. Start with a fresh install and enable the `shipping-smart-defaults` and `shipping-setting-tour` feature flags
2. Start OBW and choose `United States` as store country
3. Choose "Physical products" 
4. Complete the OBW without installing anything from the `Business Details` tab.
5. Navigate to WooCommerce -> Settings -> Shipping
6. "United States (US)" zone should be created with `Free shipping` method
7. There should be a partial tour kit spotlight over the two leftmost columns for step 1 of the tour, and over the rightmost column for step 2 of the tour.
8. Check that proceeding through the tour works as expected, and the spotlight should stick to the cells even through resizing or scrolling the window.

https://user-images.githubusercontent.com/27843274/178194400-2c7d7b41-0db3-4900-9041-8f1dfcb6caac.mp4

If you need to re-activate the tour after dismissing it, delete the option `woocommerce_admin_reviewed_default_shipping_zones` using WCA test helper.

### Other information:




### FOR PR REVIEWER ONLY:
